### DISCUSSION TITLE
Gemfile problem

### DISCUSSION THREAD

**Discussion by burninghering** [MrGreensWorkshop/MrGreen-JekyllTheme#170] [Problems] [answered]
![image](https://user-images.githubusercontent.com/37091602/217032599-b19abafa-a886-4338-a5f4-cd9b3f7e17b8.png)
Github actions give me these issues. can you give me a solution?

**Comment by MrGreensWorkshop** [comment, marked as answer]
Thanks for pointing that out. There was a problem on workflows. I fixed it now. Please update your fork.
